claim/netdata-claim.sh.in

```diff
@@ -142,6 +144,10 @@ NETDATA_USER=$(get_config_value netdata global "run as user")
 # get the MACHINE_GUID by default
 if [ -r "${MACHINE_GUID_FILE}" ]; then
         ID="$(cat "${MACHINE_GUID_FILE}")"
+        MGUID=$ID
+else
+        echo >&2 "netdata.public.unique.id is not generated yet. Please run agent at least once before attempting to claim. Agent generates this file on first startup."
```

Comment:
This means that the file is not reable. It's not 100% guaranteed that the file is not generated yet. Maybe change the language from `is not generated yet` to `may not have been generated yet`